### PR TITLE
YUMMY-001 Initial project structure

### PR DESCRIPTION
**Pull request by manuelortiz28**
Create project with pmd, check style, findbugs y crashlytics. Including
app compat

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                xmlns:tools="http://schemas.android.com/tools"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                tools:context=".MainActivity">
```
Comment (htorrex) [inline app/src/main/res/layout/activity_main.xml]: what is the advantage to specify this?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+local.properties
+gradlew
+gradlew.bat
+*.iml
+crashlytics.properties
+fabric/releaseNotes.txt
+META-INF\MANIFEST.MF
+*.project
+*.classpath
+gradle.properties
```
Comment (htorrex) [inline .gitignore]: What do you think if we add this on the file?

# built application files
*.apk

# generated files
bin/
gen/

# Local configuration file (sdk path, etc)
local.properties

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:23.+'
```
Comment (htorrex) [inline app/build.gradle]: can we avoid pointing to dynamic versions please?
reference: http://goo.gl/LvJRw7

File: app/src/main/java/com/visiontech/yummysmile/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Fabric.with(this, new Crashlytics());
+        setContentView(R.layout.activity_main);
+
+        Toolbar toolbar = ((Toolbar) findViewById(R.id.app_bar));
```
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/MainActivity.java]: :+1:

File: .gitignore (hunk 2 above)
Comment (manuelortiz28) [inline .gitignore]: *.apk are under build folder, we already have build folder
we already have local.properties

File: app/build.gradle (hunk 3 above)
Comment (manuelortiz28) [inline app/build.gradle]: :+1:

**Pull request closed without merging**